tutorials/input_and_output/input_and_output.md

```diff
@@ -134,7 +135,7 @@ To remove water molecules, we use the option `-ignore_waters`
 -->
 
 ####Zero Occupancy
-Occupancy denotes the fraction of cases where a particular conformation is observed. While most atoms will have an occupancy of 1, if a residue was observed in multiple conformations, the occupancy will be lower than 1. An occupancy of 0 indicates that the atom was never observed in the crystal (but is estimated to be present at that location). Rosetta ignores these atom records. If it is a non-backbone heavy atom, it might build the sidechain for you. If it is a backbone heavy atom like N or CA, it will delete the entire residue.
+Occupancy denotes the fraction of cases where a particular conformation is observed. While most atoms will have an occupancy of 1 (they appear in every conformation), if a residue was observed in only some conformations, the occupancy will be lower than 1. An occupancy of 0 indicates that the atom was never observed in the crystal, but is estimated to be present at that location. Rosetta ignores these atom records. If it is a non-backbone heavy atom, it might build the sidechain for you. If it is a backbone heavy atom like N or CA (the alpha-carbon of a residue), it will delete the entire residue.
```

Comment:
This may be a bit confusing, as "conformation" could mean both "state seen in the crystal" as well as "particular configuration of sidechain atoms". As I read it, the first and second sentences in this line switch between the two definitions. Might want to go with something like "Occupancy denotes the fraction of cases where _the atom was observed in those particular coordinates._ While most atoms will have an occupancy of 1 (they appear _in the same location for every copy of the protein in crystal_), if a residue was observed _in multiple locations across the multiple protein copies in the crystal_, the occupancy will be lower than 1.